tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1648,45 +1704,35 @@ def test_continuous_deletes_during_pod_restart(self):
         assert_utils.assert_equal(len(buckets), wr_bucket, f"Failed to create {wr_bucket} number "
                                                            f"of buckets. Created {len(buckets)} "
                                                            f"number of buckets")
-        LOGGER.info("Step 5: Successfully performed WRITEs with variable object sizes. (0B - "
-                    "128MB)")
-
-        LOGGER.info("Step 6: Verify %s has %s buckets created", self.s3acc_name, wr_bucket)
+        LOGGER.info("Step 3: Successfully performed WRITEs with variable object sizes.")
+        LOGGER.info("Step 4: Verify %s has %s buckets created", self.s3acc_name, wr_bucket)
         buckets = s3_test_obj.bucket_list()
         assert_utils.assert_equal(wr_bucket, len(buckets[1]), buckets)
-        LOGGER.info("Step 6: Verified %s has %s buckets created", self.s3acc_name, wr_bucket)
-
-        LOGGER.info("Step 7: Start Continuous DELETEs in background on %s buckets", del_bucket)
-        args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
+        LOGGER.info("Step 4: Verified %s has %s buckets created", self.s3acc_name, wr_bucket)
+        LOGGER.info("Step 5: Start Continuous DELETEs in background on %s buckets", del_bucket)
+        args = {'test_prefix': self.test_prefix_deg, 'test_dir_path': self.test_dir_path,
```

Comment:
we need to pass the bucket list too
```suggestion
        get_random_buck = self.system_random.sample(buckets, del_bucket)
        args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
                'skipput': True, 'skipget': True, 'bkt_list': get_random_buck,
                'output': del_output, 'bkts_to_del': del_bucket}
```